### ISSUE TITLE
include Speaker in "include agenda" of email notifier

### ISSUE THREAD

**Issue by lukasheinrich** [indico/indico#4305]
Currently we can auto generate a email notifier for an event but there seems to not be an option to include the speaker when the agenda is included. This seems like a valuable thing for people receiving the email.

**Comment by ThiefMaster** [comment]
it would also make it much less concise... so if at all it should be behind a separate option

**Comment by lukasheinrich** [comment]
I agree it could be optional, but it would make it easier for a recipient to judge whether to attend the meeting or not (perhaps just the title is not sufficient)
